test/selectors-text.spec.ts

```diff
@@ -94,6 +94,9 @@ it('query', async ({page, isWebKit}) => {
   });
   expect(await page.$eval(`text=lowo`, e => e.outerHTML)).toBe('<div>helloworld</div>');
   expect(await page.$$eval(`text=lowo`, els => els.map(e => e.outerHTML).join(''))).toBe('<div>helloworld</div><span>helloworld</span>');
+
+  await page.setContent('<button id="testAttribute">Sign&nbsp;in</button>');
```

Comment:
Do we now match '\n' as well? Let's add a test. Also, do we want to collapse spaces, e.g. match "\n \n" by a single space? Maybe only in the relaxed mode without quotes?